website/docs/usage/adding-commits.md

```diff
@@ -27,3 +27,5 @@ git commit -m "$commit_msg"
 ```
 
 The commit SHA will be empty as default when `--with-commit` is used. Specify the hash with a message separated by single whitespace for setting the commit SHA. e.g. `--with-commit "8f55e69eba6e6ce811ace32bd84cc82215673cb6 feat: add X"`
+
+Custom commits will have no effects on the commit range.
```

Comment:
This is a bit unclear for the reader - maybe update it like:

```
:::info

 These added commits will have empty [`commit_range`](docs/etc/etc) variable etc.

::: 
```

Or something along those lines.